src/status_im2/contexts/wallet/common/temp.cljs

```diff
@@ -102,3 +103,28 @@
                         :alignment   :flex-start}
     :description       :text
     :description-props {:text (string/replace constants/path-default-wallet #"/" " / ")}}])
+
+(def network-names [:ethereum :optimism :arbitrum])
+
+(def address "0x39cf6E0Ba4C4530735616e1Ee7ff5FbCB726fBd4")
```

Comment:
what address is this btw? any chance we can a bit more verbose as there are many different types of addresses

Reply:
@J-Son89 this is copied from the design. This is probably the account address (which ofc comes from the backend)